fe/fe-core/src/main/java/com/starrocks/http/rest/LoadAction.java

```diff
@@ -100,9 +101,13 @@ public void executeWithoutPassword(BaseRequest request, BaseResponse response) t
             throw new DdlException("No backend alive.");
         }
 
-        Backend backend = GlobalStateMgr.getCurrentSystemInfo().getBackend(backendIds.get(0));
+        // TODO: need to refactor after be split into cn + dn
+        ComputeNode backend = GlobalStateMgr.getCurrentSystemInfo().getBackend(backendIds.get(0));
+        if (RunMode.getCurrentRunMode() == RunMode.SHARED_DATA) {
```

Comment:
can we trust the backendId chosen previously, only ensure that it can be found either as ComputeNode or BackendNode, no matter what run mode it is right now?